client/data/disputes/actions.js

```diff
@@ -88,3 +89,58 @@ export function* acceptDispute( id ) {
 		yield controls.dispatch( 'core/notices', 'createErrorNotice', message );
 	}
 }
+
+// This function handles the dispute acceptance flow from the Transaction Details screen.
+// It will become the default acceptDispute function once the feature flag
+// '_wcpay_feature_dispute_on_transaction_page' is enabled by default.
+export function* acceptTransactionDetailsDispute( dispute ) {
```

Comment:
Rather than modify the existing `acceptDispute` action (used on the Dispute → Details screen), I've created a new one since there are some differences in the approach:
1. The existing action will force a redirect to Payments → Disputes.
2. This new action requires fetching and updating the payment intent to refresh the current Transaction Details page.

Once the feature flag is lifted, `_wcpay_feature_dispute_on_transaction_page`, we can remove and replace the previous action with the new one.